pywbem/cim_obj.py

```diff
@@ -1928,29 +1928,40 @@ def to_wbem_uri(self, format='standard'):
         The returned WBEM URI contains its components as follows:
 
         * it does not contain a namespace type (URI scheme).
-        * it contains an authority component according to the
+        * it contains an authority (host) component according to the
           :attr:`~pywbem.CIMInstanceName.host` attribute, if that is not
-          `None`. Othwerise, it does not contain the authority component.
+          `None`. Otherwise, it does not contain the authority component.
         * it contains a namespace component according to the
           :attr:`~pywbem.CIMInstanceName.namespace` attribute, if that is not
-          `None`. Othwerise, it does not contain the namespace component.
+          `None`. Otherwise, it does not contain the namespace component.
         * it contains a class name component according to the
           :attr:`~pywbem.CIMInstanceName.classname` attribute.
         * it contains keybindings according to the
           :attr:`~pywbem.CIMInstanceName.keybindings` attribute, with the
           order of keybindings preserved, and the lexical case of keybinding
-          names preserved.
-
-        :term:`DSP0004` defines instance paths without a concept of order in
-        their keybindings, and that keybinding names in instance paths match
-        case-insensitively. Because the WBEM URI string returned by this method
-        preserves the order of keybindings (relative to how the keybindings
-        were first added to this object) and because the lexical case of the
-        keybinding names is also preserved, equality of WBEM URIs should *not*
-        be determined by comparing the returned WBEM URI strings. Instead,
-        compare :class:`~pywbem.CIMInstanceName` objects using the ``==``
-        operator, which performs the comparison at the logical level required
-        by :term:`DSP0004`. If you have WBEM URI strings without the
+          names preserved (except when using the format "canonical").
+
+        Note that when you do not want some of these components to show up
+        in the resulting WBEM URI string, you can set them to `None` before
+        calling this method.
+
+        Except when using the format "canonical", this method should not be
+        used to compare instance paths for equality: :term:`DSP0004` defines
```

Comment:
How about adding 'or using paths as dictionary keys' in particular since that was Jerry's example of the issue.